docs/content/concepts/graphql-rpc.mdx

```diff
@@ -25,6 +25,15 @@ For more details on GraphQL fundamentals, see the introductory documentation fro
 
 :::
 
+:::caution
+
+Switch any apps that still use the GraphQL Alpha endpoints (`https://sui-mainnet.mystenlabs.com/graphql` and `https://sui-testnet.mystenlabs.com/graphql`) to the GraphQL Beta endpoints as soon as possible to avoid disruption of service.
+
+- GraphQL Beta Mainnet: https://graphql.mainnet.sui.io/graphql
+- GraphQL Beta Testnet: https://graphql.testnet.sui.io/graphql
```

Comment:
Chris very kindly set-up the devnet URL as well:

```suggestion
- GraphQL Beta Testnet: https://graphql.testnet.sui.io/graphql
- GraphQL Beta Devnet: https://graphql.devnet.sui.io/graphql
```

Reply:
And I guess we should mention `https://sui-devnet.mystenlabs.com/graphql` as one of the alpha URLs, in the paragraph above.

Reply:
The `curl` call in the "Headers" section below also needs to be updated with the new URL, and the output regenerated with the beta service -- there may be other similar cases as well.

There are also queries embedded in those `curl` calls that need to be tweaked. E.g. in the "Variables" section, the query should look like this:

```sh
curl -X POST https://graphql.testnet.sui.io/graphql \
    --header 'Content-Type: application/json'       \
    --data '{
        "query": "query ($id: Int) { epoch(epochId: $id) { referenceGasPrice } }",
        "variables": { "id": 100 }
    }'
```

Reply:
Ditto for the Guide page